<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenerixProjectPageForm</class>
 <widget class="QWidget" name="GenerixProjectPageForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>General</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Webmodule location :</string>
     </property>
     <property name="buddy">
      <cstring>m_webModuleLocation</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="DirectoryEditWidget" name="m_webModuleLocation" native="true">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="directory" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Property file :</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="m_propertyFileLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Version :</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="ConfigurationVersionLabel" name="m_configurationVersionLabel"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Number of dictionary :</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="m_dictionaryCount">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Number of businessview :</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="m_businessViewLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DirectoryEditWidget</class>
   <extends>QWidget</extends>
   <header>directoryedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConfigurationVersionLabel</class>
   <extends>QLabel</extends>
   <header>configuration/configurationfileedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
